<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<HTML>
<HEAD>
	<TITLE>The Migration History of Humans</TITLE>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
 <link href="pnnbook.css" rel="stylesheet" type="text/css" charset="utf-8">
</HEAD>

<BODY bgcolor="#dafeec" >

<table border="0" cellspacing="2" cellpadding="2" align="center">
<tr class="whead">

    <td>The Migration History of Humans: DNA Study Traces Human Origins Across the
Continents<br>DNA furnishes an ever clearer picture of the multimillennial trek from Africa all
the way to the tip of South America

By Gary Stix </td></tr></table>


<table cellspacing="2" cellpadding="2" border="0">
<tr class="w2text">
<td>
<div align="center"><h4>Key Concepts</h4></div>
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    * Scientists trace the path of human migrations by using bones, artifacts and DNA. Ancient
objects, however, are hard to find.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    * DNA from contemporary humans can be compared to determine how long an indigenous
population has lived in a region.

<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    * The latest studies survey swathes of entire genomes and produce maps of human movements
across much of the world. They also describe how people’s genes have adapted to changes in
diet, climate and disease.


</td></tr>
<tr class="w2text">
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;A development company controlled by Osama bin Laden’s
half brother revealed last year that it wants to build a bridge that will span the Bab el
Mandeb, the outlet of the Red Sea to the Indian Ocean. If this ambitious project is ever
realized, the throngs of African pilgrims who traverse one of the longest bridges in the world
on a journey to Mecca would pass hundreds of feet above the probable route of the most
memorable journey in human history. Fifty or sixty thousand years ago a small band of
Africans—a few hundred or even several thousand—crossed the strait in tiny boats, never to
return.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
The reason they left their homeland in eastern Africa is not completely understood. Perhaps the
climate changed, or once abundant shellfish stocks vanished. But some things are fairly
certain. Those first trekkers out of Africa brought with them the physical and behavioral
traits—the large brains and the capacity for language—that characterize fully modern humans.
From their bivouac on the Asian continent in what is now Yemen, they set out on a
decamillennial journey that spanned continents and land bridges and reached all the way to
Tierra del Fuego, at the bottom of South America.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Scientists, of course, have gained insight into these wanderings because of the fossilized
bones or spearheads laboriously uncovered and stored in collections. But ancestral
hand-me-downs are often too scant to provide a complete picture of this remote history. In the
past 20 years population geneticists have begun to fill in gaps in the paleoanthropological
record by fashioning a genetic bread-crumb trail of the earliest migrations by modern humans.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Almost all our DNA—99.9 percent of the three billion “letters,” or nucleotides, that make up
the human genome—is the same from person to person. But interwoven in that last 0.1 percent are
telltale differences. A comparison among, say, East Africans and Native Americans can yield
vital clues to human ancestry and to the inexorable progression of colonizations from continent
to continent. Until recent years, DNA passed down only from fathers to sons or from mothers to
their children has served as the equivalent of fossilized footprints for geneticists. The
newest research lets scientists adjust their focus, widening the field of view beyond a few
isolated stretches of DNA to inspect hundreds of thousands of nucleotides scattered throughout
the whole genome.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Scanning broadly has produced global migratory maps of unprecedented resolution, some of which
have been published only during recent months. The research provides an endorsement of modern
human origins in Africa and shows how that continent served as a reservoir of genetic diversity
that trickled out to the rest of the world. A genetic family tree that begins with the San
people of Africa at its root ends with South American Indians and Pacific Islanders on its
youngest-growing branches.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
The study of human genetic variation—a kind of historical Global Positioning System—goes back
to World War I, when two physicians working in the Greek city of Thessaloni'ki found that
soldiers garrisoned there had a differing incidence of a given blood group depending on their
nationality. Beginning in the 1950s, Luigi Luca Cavalli-Sforza started formalizing the study of
genetic differences among populations by examining distinct blood group proteins. Variations in
proteins reflect differences in the genes that encode them.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

Then, in 1987, Rebecca L. Cann and Allan C. Wilson of the University of California, Berkeley,
published a groundbreaking paper based on analyzing the DNA of mitochondria, the cell’s
energy-producing organelles, which are passed down through the maternal line. They reported
that humans from different populations all descended from a single female in Africa who lived
about 200,000 years ago—a finding that immediately made headlines trumpeting the discovery of
the “Mitochondrial Eve.” (Despite the Biblical allusion, this Eve was not the first woman: her
lineage, though, is all that has survived.)
</td></tr>
<tr class="w2text">
<td><div align="center"><h4>All about Eve</h4></div>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
The fast, relatively predictable rate of “neutral” mitochondrial mutations—ones that are
neither beneficial nor harmful—lets the organelles operate as molecular clocks. Counting the
differences in the number of mutations (ticks of the clock) between two groups, or lineages,
allows a researcher to construct a genetic tree that tracks back to a common
ancestor—Mitochondrial Eve or another woman who founded a new lineage. Comparison of the ages
of the lineages from different regions permits the building of a timeline of human migrations.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Since 1987 the data bank on human diversity has broadened to encompass the Y chromosome—the sex
chromosome passed down only by males to their sons. The male-transmitted DNA carries many more
nucleotides than mitochondrial DNA does (tens of millions, as opposed to just 16,000),
enhancing investigators’ ability to distinguish one population from another. Analyzing
mitochondrial and Y chromosome DNA from human populations has turned up hundreds of genetic
markers (DNA sites having identifiable mutations specific to particular lineages).
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
The route humans took from Africa to the Americas over the course of tens of thousands of years
can now be tracked on the map as if the travelers were moving, albeit extremely slowly, on a
series of interconnected superhighways. Alphanumeric route signs, such as I-95, can be recast
as alphanumeric genetic markers. In the case of the Y chromosome, for instance, cross the Bab
el Mandeb on highway (genetic marker) M168, which becomes M89 when heading north through the
Arabian Peninsula. Make a right at M9 and set out toward Mesopotamia and beyond. Once reaching
an area north of the Hindu Kush, turn left onto M45. In Siberia, go right and follow M242 until
it eventually traverses the land bridge to Alaska. Pick up M3 and proceed to South America.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Mitochondrial DNA and the Y chromosome remain powerful analytical instruments. The National
Geographic Society, IBM and the Waitt Family Foundation have joined in a privately funded
$40-million collaboration through 2010, research that is primarily devoted to using these
tools. With the help of 10 regional academic institutions, the so-called Genographic Project is
gathering DNA from up to 100,000 indigenous people worldwide. “What we’re focusing on is the
details of how people made the journeys,” says Spencer Wells, who heads the project. In a
recent report its researchers found that the Khoisan people of southern Africa remained
genetically separate from other Africans for 100,000 years. In another study, they demonstrated
that some of the gene pool of Lebanese men can be traced to Christian Crusaders and Muslims
from the Arabian Peninsula.
<br><br>
<div align="center"><h4>Power Tools</h4></div>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

Genetic researchers have sampled the DNA of many people living along the migratory routes they
have discovered. Yet the seeming certainty of the data sometimes deceives. Scientists who study
human origins still would prefer a fossil they can hold in their hands over a genealogical
tree. DNA differs from the radioactive isotopes used to date fossils. The rate of mutation can
fluctuate from one stretch of DNA to another.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
But paleoanthropologists are in a fix. Fossil remains are rare and too often incomplete. The
earliest migration from Africa to Australia shows up in mitochondrial and Y genetic material
(thanks to Andaman Islanders, among others), but the physical artifacts are largely missing
along the route.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
The answer to the absence of stones and bones: more DNA, from wherever. To bolster the case for
genetics, researchers have looked to microbes that have hitched a ride on humans, inspecting
their genes to look for similar patterns of migration. Freeloaders include bacteria, viruses
and even lice. Besides microorganisms, the Human Genome Project and related efforts to look
across the expanse of whole genomes have yielded a set of power tools that are helping to
compensate for deficiencies in genetic methods. “You can look at so many different places in
the genome from many individuals and in many populations to achieve more statistical power in
testing different hypotheses,” says Tim Weaver, a professor of anthropology at the University
of California, Davis.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
During this decade, researchers have made dramatic discoveries by simultaneously comparing a
multitude of variable, or polymorphic, sites interspersed throughout the genome’s three billion
nucleotides. The first whole-genome studies earlier in this decade looked at differences among
populations in short repetitive stretches of DNA known as microsatellites. More recently, the
scope afforded by whole-genome scans has widened further. In February two papers, one in
Science, the other in Nature, reported the largest surveys to date of human diversity. Both
examined more than 500,000 single nucleotide polymorphisms (SNPs)—swaps of one nucleotide for
another at a particular spot in the DNA—from the Human Genome Diversity Panel. These cell lines
were drawn from about 1,000 individuals from 51 populations worldwide and are maintained by the
Center for the Study of Human Polymorphisms in Paris.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
The two research teams analyzed the wealth of data in various ways. They compared SNPs directly
among distinct populations. They also looked at haplotypes, blocks of DNA containing numerous
SNPs that are inherited intact through many generations. The group that wrote the Nature paper
also explored a new technique for surveying human variation by comparing repetitions or
deletions of DNA stretches of up to 1,000,000 nucleotides long (copy number variations)
throughout a person’s genome, consistent with the larger trend to mine the genome for ever more
markers of variation. “Any one piece of the genome will have a history that doesn’t necessarily
reflect the ancestry of the genome as a whole,” says Noah A. Rosenberg of the University of
Michigan at Ann Arbor and lead author of the Nature paper. But looking at many areas at once,
he explains, can overcome that problem: “With thousands of markers, it’s possible to determine
the overall story of human migrations.”
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Looking at hundreds of thousands of SNPs allowed the researchers to resolve the identities of
individual populations—and to see how genetically close relations spread far and wide. Native
South American ancestry was tracked back to Siberians and some other Asians. The Han people,
China’s principle ethnic group, has distinct northern and southern populations. Bedouins are
related to groups from Europe and Pakistan as well as the Middle East.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
The findings, which jibed with previous research from anthropology, archaeology, linguistics
and biology (including previous mitochondrial and Y DNA studies), also provided a broader
statistical foundation for the out-of-Africa hypothesis, supporting the idea that a small
population of humans moved out of the continent, then grew in size in a new home until another
subgroup of “founders” broke off and moved away—a process that repeated itself until the entire
world was settled. These wayfarers edged out archaic human populations—Homo neanderthalensis
and Homo erectus—with little or no interbreeding when they met. The new DNA work indicates that
each time a smaller group split off, it carried only a subset of the genetic diversity
originally present in the African population. So as distance (and time) removed from Africa
lengthens, diversity diminishes, providing a means to follow population movements. Native
Americans, sojourners on the last major continental migrations, have much less variety in their
genomes than Africans do.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Many scientists believe that the weight of evidence, now backed by large statistical analyses
such as the ones in Science and Nature, gives the out-of-Africa proponents a clear edge in a
long-running debate over human origins. The multiregional hypothesis—a competitor to the
out-of-Africa one—argues that populations that descended from archaics, such as H. erectus,
evolved over the past 1.8 million years in Africa, Europe and Asia, and gradually emerged as
Homo sapiens. Occasional interbreeding ensured that the groups did not split off into separate
species.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Few scientists still hold a banner for a strict interpretation of multiregionalism. But
modified versions still circulate, mostly as attempts to pinpoint whether H. sapiens bear
genetic signatures of our encounters with hominid cousins. Vinayak Eswaran of the Indian
Institute of Technology, aided by Henry C. Harpending and Alan R. Rogers of the University of
Utah, came up with a set of simulations in recent years that suggest that after humans migrated
out of Africa they interbred extensively with archaic species such as H. erectus. Eswaran’s
model suggests that as much as 80 percent of the modern human genome may have been subject to
the effects of this kind of interbreeding.

<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
The genetic imprint is not as visible as might be expected if interbreeding occurred, but
Harpending offers an explanation. A set of beneficial genes carried by African emigrants,
perhaps ones that assisted in childbearing, brought a selective advantage that eventually
blotted out the signature of some archaic genes. “The result is that the population seems more
closely related to the [African] source population of the favored genes than it really is,” he
says.
</td></tr>

<tr class="w2text">
<td><div align="center"><h4>Are We Part Neandertal?</h4></div>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Eswaran and Harpending are not the only ones suggesting the possibility of interspecies trysts.
Some fossilized skeletal remains of H. sapiens have features reminiscent of earlier hominids,
and the genetic record of contemporary humans has also provided fuel for discussion.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
According to the tree diagrams that document genetic lineages, some gene variants show “deep
ancestry”—they are much older than they should be if humans evolved from a single homogeneous
group no more than 200,000 years ago; a hint of possible interbreeding. In one study that drew
attention in 2006, Bruce T. Lahn of the University of Chicago and his colleagues reported that
a version of the Microcephalin gene, which is involved in regulating brain size, contains a
haplotype that may have been passed on during an encounter with Neandertals 40,000 years ago.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
A more definitive answer may arrive within the next 12 months. The Neandertal Genome Project—a
collaboration of the Max Planck Institute for Evolutionary Anthropology in Leipzig, Germany,
and 454 Life Sciences, a Connecticut-based sequencing company—is scheduled by the end of this
year to have finished a rough draft of some 70 percent of the sequences of DNA from
40,000-year-old Neandertal bones from a Croatian cave. Its results are expected to be published
about six months later.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
So far the project has unearthed no sign of any genetic pattern that would suggest DNA transfer
between the two hominid lineages. “We see no evidence of that, but we can’t exclude it,” says
Svante Pa"a"bo, the Max Planck professor who heads the project. An earlier publication by his
group that surveyed one million nucleotides, a minuscule fraction of the whole genome,
suggested that some gene exchange might have occurred, but the result was later found to have
been a false signal because of sample contamination. The researchers have not yet encountered
the Microcephalin variant cited by Lahn.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Handling or even breathing on a sample remains an impediment to working with ancient DNA: some
anthropologists wrap themselves in the clean-room “bunny suits” used in microchip factories
when they head to the field on a dig. Since that initial paper, Pa"a"bo’s laboratory has
altered the procedures used in the clean room at Max Planck. Researchers place tags made up of
four nucleotides of synthetic DNA at the beginning of each strand of Neandertal genetic
material. Each strand that exits the sequencing machine goes through a molecular identity
check.

<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
An understanding of the genetic makeup of the closest cousin in the human line—estimates from
previous studies show that the two genomes are about 99.5 percent alike—could provide the most
incisive exercise to date in comparative genomics, allowing identification of sites in the
human genome where interbreeding took its course and where natural selection favored certain
traits. “I think if you’re interested in human evolution, Neandertals are the unique thing,”
Pa"a"bo says. “They are our closest relatives. You can access their genomes, even though it’s
technically difficult. But for most other ancestral human groups, that will not be possible.”
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
New, still unpublished work reveals that the Neandertal Y chromosome differs from the human
one. “No human man has a Y chromosome like that of the Neandertal,” Pa"a"bo observes, mirroring
earlier results showing that human and Neandertal mitochondrial DNA also are readily
distinguishable. Last November Pa"a"bo and his team did report one similarity between the two
hominids. Neandertal remains from Spain had a version of a gene known as FOXP2 that is
identical to one in humans that is involved with the development of speech and language. Again,
speculation emerged in a paper by a separate group in April about whether the gene could have
resulted from interbreeding, although the possibility of contamination could not be discounted.
</td></tr>


<tr class="w2text">
<td><div align="center"><h4>How Have We Adapted?</h4></div>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
As researchers continue sequencing DNA from shards of old bone to explore whether humans mated
with other species of the genus Homo, other investigators are applying genome-wide analyses of
DNA to see which genetically controlled traits changed through genetic drift (random mutations)
and natural selection as migrants adapted to their new homes.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
A study published in February in Nature showed the consequences of the declining genetic
diversity as humans left Africa. The project compared 40,000 SNPs from a group of 20
European-Americans and 15 African-Americans. It found that the European-Americans had a higher
proportion of harmful genetic changes, ones potentially related to disease, than the
African-Americans did, although the authors refrained from speculation about any specific
health effects. The research shows what lead scientist Carlos D. Bustamante called a
“population genetic echo” of Europe’s founding. The low genetic diversity of Europe’s small
initial population permitted a set of harmful mutations to disperse widely and new harmful ones
to emerge when the numbers of people began to grow. Natural selection has not yet had time to
remove deleterious changes.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Genome-wide research is also starting to furnish a panoramic picture of how natural selection
helped migrants adapt to new environs. A spate of studies in the past two years have  looked
for genetic alterations that have occurred since humans left Africa or took up agriculture and
that appear to have been useful for surviving in novel circumstances. Genetic prospectors mined
the International HapMap, a catalogue of haplotypes and the 3.9 million SNPs contained therein
from North Americans with ancestry in northwestern Europe and from individuals sampled in
Nigeria, China and Japan.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

One study, co-authored by Harpending, showed that the rate of change of DNA, and thus the pace
of evolution, has accelerated over the past 40,000 years. Another by Pardis C. Sabeti of the
Broad Institute in Cambridge, Mass., and her colleagues indicated that hundreds of regions of
the genome are still undergoing selection, including areas that govern resistance to disease
and the development of skin color, and hair follicles, which  regulate sweat. Such findings
imply that human populations are continuing to adapt to regional differences in sun exposure,
foods and pathogens they encountered when they left their ancestral African home. And Africans
have also evolved as their environs changed.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
One of the most recent studies, led by Lluis Quintana-Murci of the Pasteur Institute in Paris,
showed that 580 genes, including ones that play a role in diabetes, obesity and hypertension,
are undergoing selection differently among the HapMap populations, perhaps explaining
geographical differences in disease patterns and providing clues to new targets for developing
drugs.
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Consideration of the processes underlying human diversity sometimes moves beyond the dimensions
of hair follicles and the ability to digest milk. Debate over what constitutes race and
ethnicity can quickly enter the picture. What does it mean if a gene variant related to
cognition is found more in Europeans than in Africans? Better public understanding of
genetics—that a single gene does not act like a light switch that toggles between intelligence
and doltishness—may quell misguided speculations.<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Genetic literacy will let a term like “Asian” or “Chinese” be replaced by more subtle
classifications based on the differences in ancestral genetic makeup found in recent
genome-wide scans, such as the distinction between China’s southern and northern Han groups.
“There is no race,” Quintana-Murci says. “What we see [from the standpoint of genetics] is
geographical gradients. There are no sharp differences between Europeans and Asians. From
Ireland to Japan, there is no sharp boundary where something has changed completely.”
<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
The journey through evolutionary history set in place by comparative genomics is still
starting. In the meantime, the hunger for more data and more powerful computers and algorithms
knows no limits. Amassing larger databases—an international consortium announced in January its
intention to sequence 1,000 genomes from various regional populations—will let researchers run
ever more realistic simulations of alternative models of human evolution and weigh the
probabilities of each one, yielding the best picture yet of who we are and where we came from.
<br><br>
Note: This story was originally published with the title, "Traces of a Distant Past".
</td></tr>

<tr class="w2text">
<td>
</td></tr>

</table>


</body>
</html>
